src/sidebar/services/autosave.ts

```diff
@@ -65,9 +66,12 @@ export class AutosaveService {
 
         retryPromiseOperation(() => this._annotationsService.save(highlight))
           .then(() => {
-            this._toastMessenger.success('Highlight saved', {
-              visuallyHidden: true,
-            });
+            this._toastMessenger.success(
+              `Highlight saved for "${quote(highlight)}"`,
```

Comment:
```suggestion
              `Highlighted "${quote(highlight)}"`,
```

To me this sounds slightly more natural than "Highlight saved for...".

Reply:
On a related note, I think we should remove the "Success" phrase from the start of hidden toast notification messages, as this typically ends up duplicating information in the message ("Highlighted", "Annotation saved").

Reply:
> On a related note, I think we should remove the "Success" phrase from the start of hidden toast notification messages,

Agreed